calmap/__init__.py

```diff
@@ -16,6 +16,8 @@
 import numpy as np
 import pandas as pd
 from distutils.version import StrictVersion
+from dateutil.relativedelta import relativedelta
+from matplotlib.patches import Polygon
 
 __version_info__ = ("0", "0", "8")
```

Comment:
can you update version info to 0 0 9

Reply:
```suggestion
__version_info__ = ("0", "0", "9")
```

Here's a one-click apply for you.